services/minespace-web/common/constants/strings.js

```diff
@@ -76,6 +76,7 @@ export const INCIDENT_DETERMINATION_TYPES = {
 export const INCIDENT_FOLLOWUP_ACTIONS = {
   miu: "MIU",
   inspector: "INS",
+  projectLead: "PRL",
   none: "NO",
```

Comment:
I don't think this needs to be here - This is for incidents and they do not have a projectLead, only Inspectors.